src/utils/ui.js

```diff
@@ -73,7 +74,7 @@ export function hexToRgba(hex: string, opacity: ?number) {
     : null;
 }
 
-export const measure = (ref: View): Promise<Measurements> =>
-  new Promise(resolve => ref.measureInWindow((x, y, w, h) => resolve({
+export const measure = (ref: ?React.ElementRef<typeof View>): Promise<Measurements> =>
+  new Promise(resolve => ref?.measureInWindow((x, y, w, h) => resolve({
```

Comment:
I'm not sure if `ref` being set to `undefined` actually occurs in practice, but if it does then the returned promise will hang where previously it would throw. Looking at usages of `measure`, nothing would change, but rejected promises are expected, so it's probably best to return one in that case for consistency.

Reply:
removed nullable check